UIkitDZ/VC/PizzaIngridients.swift

```diff
@@ -0,0 +1,137 @@
+        dismiss(animated: true, completion: nil)
+    }
+    
+    func createButtonPrint(x: Int, y: Int) {
+        let buttonPlus = UIButton()
+        buttonPlus.frame = CGRect(x: x, y: y, width: 40, height: 40)
+        buttonPlus.setBackgroundImage(UIImage(systemName: "plus.app.fill"), for: .normal)
+        buttonPlus.tintColor = .orange
+        buttonPlus.addTarget(self, action: #selector(showPayment), for: .touchUpInside)
+        self.view.addSubview(buttonPlus)
+    }
+    
+    @objc func showPayment() {
+        let paymentViewController = PaymentVC()
+        let paymentNavigationController = UINavigationController(rootViewController: paymentViewController)
+        paymentNavigationController.modalPresentationStyle = .fullScreen
+        if cheeseDolly.isOn {
+            paymentViewController.cheese = true
+        }
+        if bakonDolly.isOn {
+            paymentViewController.bacon = true
+        }
+        if mashroomDolly.isOn {
+            paymentViewController.mashrooms = true
+        }
+        if oliveDolly.isOn {
+            paymentViewController.olive = true
+        }
+        paymentViewController.delegate = self
+        paymentViewController.nameOfPizza = namePizzaStr
+        present(paymentNavigationController, animated: true, completion: nil)
+        
+    }
+    
+    func createPizzaImage(img: String, x: Int, y: Int) {
+        let pizzaImageView = UIImageView()
+        pizzaImageView.image = UIImage(named: img)
+        pizzaImageView.frame = CGRect(x: x, y: y, width: 250, height: 250)
+        pizzaImageView.backgroundColor = UIColor.clear
+        self.view.addSubview(pizzaImageView)
+    }
+    
+    func createPizzaLabel(nameType: String, x: Int, y: Int, color: UIColor, size: CGFloat) {
+        let namePizza = UILabel()
+        namePizza.text = nameType
+        namePizza.frame = CGRect(x: x, y: y, width: 250, height: 60)
+        namePizza.backgroundColor = UIColor.clear
+        namePizza.textColor = color
+        namePizza.font = namePizza.font.withSize(size)
+        self.view.addSubview(namePizza)
+    }
+    
+    func createSwitch(x: Int, y: Int, tag: Int) -> UISwitch {
+        let swSwitch = UISwitch()
+        swSwitch.frame = CGRect(x: x, y: y, width: 30, height: 30)
+        swSwitch.tag = tag
+        self.view.addSubview(swSwitch)
+        switch tag {
+        case 0:
+            return swSwitch
+        case 1:
+            return swSwitch
+        case 2:
+            return swSwitch
+        case 3:
+            return swSwitch
+        default:
+            return swSwitch
+        }
+    }
+}
+
+extension PizzaIngridients: getToLoggedVC {
+    func getToLogged() {
+        if let currentNVC = presentingViewController as? UINavigationController {
+            self.view.isHidden = true
+            self.dismiss(animated: false, completion: nil)
+            currentNVC.popToRootViewController(animated: false)
+        }
+    }
+}
+
+protocol getToLoggedVC: AnyObject {
```

Comment:
```suggestion
protocol GetToLoggedVCDelegate: AnyObject {
```
с маленькой буквы серьезно?